tests/test_data/dummy_bootloaders/bootloader2

```diff
@@ -0,0 +1 @@
+i am fake bootloader data
```

Comment:
Nothing you did is wrong, however this woud be the python way of doing it: https://docs.pytest.org/en/6.2.x/tmpdir.html